src/Utils/FeaturesUtils.js

```diff
@@ -145,39 +147,37 @@ export default {
     filterFeaturesUnderCoordinate(coordinate, features, epsilon = 0.1) {
         const result = [];
 
+        coord.crs = coordinate.crs;
```

Comment:
`Coordinates.copy` should be copy already the crs, no? WDYM?

Reply:
Yes, that's something I saw and wanted to do ! Updating now